docs/en/guides/developer/working-with-json/json-semi-structured.md

```diff
@@ -22,30 +22,34 @@ It is important to note that the JSON type primarily syntactically enhances JSON
 Note that recent versions of ClickHouse (22.4.1+) will infer the schema for JSONEachRow. This inference will also work for JSON objects with nested structures. These will be inferred as JSON object fields. For example, executing a DESCRIBE shows the detected schema for the file, including the actor fields:
 
 ```sql
-DESCRIBE s3('https://datasets-documentation.s3.eu-west-3.amazonaws.com/github/github-2022.ndjson.gz', 
-'JSONEachRow') SETTINGS input_format_max_rows_to_read_for_schema_inference=100;
+DESCRIBE s3('https://datasets-documentation.s3.eu-west-3.amazonaws.com/github/github-2022.ndjson.gz',
+'JSONEachRow')
+SETTINGS input_format_max_rows_to_read_for_schema_inference = 100,
+input_format_json_read_objects_as_strings = 1
 ```
 
 | name | type | 
 | :--- | :--- | 
 | type | Nullable\(String\) |
-| actor | Object\('json'\) |
-| repo | Object\('json'\) |
+| actor | Nullable(String) |
+| repo | Nullable(String) |
 | created\_at | Nullable\(String\) |
-| payload | Object\('json'\) |
+| payload | Nullable(String) |
```

Comment:
I still think we should leave it as it was before, but add setting `allow_experimental_object_type=1` and then add same queires with `input_format_json_read_objects_as_strings` to show the user that this dataset can be loaded without using experimental JSON type.

Reply:
agreed, I will update once I finish the json-load-data